mautic/mautic
# Salesforce contact link

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR puts the contact timeline link in a SF field if matched through the plugin UI
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Apply PR, match the contact timeline link with a Salesforce url field in the plugin config modal window, (the Salesforce fields can be a custom url field)
<img width="588" alt="screen shot 2017-04-10 at 18 20 41" src="https://cloud.githubusercontent.com/assets/1496976/24873910/8e29c8d6-1e1a-11e7-8b27-61538f476d72.png">

2. push a contact either through a push to integration action or the  cron job, and you should see this link in the SF lead or contact
